bsp4j/src/main/xtend-gen/ch/epfl/scala/bsp4j/JvmEnvironmentItem.java

```diff
@@ -21,14 +21,18 @@ public class JvmEnvironmentItem {
   @NonNull
   private String workingDirectory;
 
+  @NonNull
```

Comment:
Making this required break backwards compat, and since some implementations already rely on this we shouldn't without very good reason